src/utils/localise.ts

```diff
@@ -50,3 +48,11 @@ export function getLocalesField(fieldName: string, req: Request): string {
     }
 
 }
+
+export function getLanguageChoice(req: Request): string {
+    const QUERY_LANG = "lang";
```

Comment:
Do we need this as a separate variable, could we not just pass the string straight to the function